pages/common/plantuml.md

```diff
@@ -0,0 +1,28 @@
+# plantuml
+
+> Create UML diagrams from a plain text language and render them in different formats.
+> More information <https://plantuml.com/en/command-line>.
+
+- Render diagrams to default format (PNG):
+
+`plantuml {{ diagram1.puml }} {{ diagram2.puml }}`
+
+- Render diagram in given format (e.g. `png`, `pdf`, `svg`, `txt`):
+
+`plantuml -t{{ format }} {{ diagram.puml }}`
```

Comment:
```suggestion
`plantuml -t {{format}} {{diagram.puml}}`
```

Reply:
This change would be wrong because PlantUML really is inflexible. One _must_ use `-tpng` or `-tsvg` etc. without any whitespace _e.g._:
```bash
plantuml -tsvg diagram.puml
```
Will produce a `diagram.svg` file. However
```bash
plantuml -t svg diagram.puml
```
Will produce a `diagram.png` file (and will not even output errors nor warning).

Reply:
@michaeldel Oh, alright. I implemented the change to respect that property as well.